### DISCUSSION TITLE
Page not Found; 404 being thrown

### DISCUSSION THREAD

**Discussion by elicorlette** [community/community#22629] [Other Feature Feedback, Questions, & Ideas]
<p>I have a very simple setup right now as far as code goes as I just want to make sure it will actually show up, but I’m not sure why I’m getting an error.</p>
<p>Site: <a href="https://elicorlette.com/" rel="nofollow noopener">https://elicorlette.com/</a></p>
<p>Repo: <a href="https://github.com/elicorlette/elicorlette.github.io" rel="nofollow noopener">https://github.com/elicorlette/elicorlette.github.io</a></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I opened <a href="https://github.com/elicorlette/elicorlette.github.io/pull/1" rel="nofollow noopener">a pull request</a> for you to fix the typo you have in the index.html file name :grinning:</p>

**Comment by elicorlette** [comment]
<p>I am so mad, I’ve been trying to figure out the problem for so long and it was such a dumb error, thank you!:smileyvery-happy:</p>
